.gitignore

```diff
@@ -107,3 +107,7 @@ ENV/
 /lib/
 pip-selfcheck.json
 pyvenv.cfg
+
+checkfiles/.DS_Store
```

Comment:
change 
```
checkfiles/.DS_Store
```
to
```
.DS_Store
```

Reply:
I am not sure about this one, @strattan would know what to add to .gitignore for DNanexus files. Currently, .gitignore supposed to ignore both the repo main directory and the internal checkfiles directory for .DS_Store